cdap-ui/cypress/fixtures/missing-artifact-pipeline.json

```diff
@@ -0,0 +1,149 @@
+{
+    "artifact": {
+        "name": "cdap-data-pipeline",
+        "version": "6.1.3-SNAPSHOT",
```

Comment:
Probably want a version range, i.e. [6.1.3-SNAPSHOT, 6.3.0-SNAPSHOT]

Reply:
Fixed it